fedora/client/openidbaseclient.py

```diff
@@ -129,7 +133,9 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
         :kwarg timeout: A float describing the timeout of the connection. The
             timeout only affects the connection process itself, not the
             downloading of the response body. Defaults to 120 seconds.
-
+        :kwarg retry_backoff_factor: Backoff factor to apply between attempts.
+            Sleep for `{backoff factor}*(2 ^ ({number of total retries} - 1))`
```

Comment:
This only makes sense to me if I change "number of total retries" to "number of attempted retries" (or "failed retries", should be the same). I know it's been copy pasted from upstream documentation, but I think they have it written in a confusing way, because "total" seems to be the equivalent of "max", and that's a different number. Just a note.